include/RED4ext/CommandLine-inl.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/CommandLine.hpp>
+#endif
+
+#include <RED4ext/CString.hpp>
+#include <RED4ext/Detail/AddressHashes.hpp>
+#include <RED4ext/Relocation.hpp>
+
+RED4EXT_INLINE const RED4ext::CommandLine* RED4ext::CommandLine::Get() noexcept
+{
+    // Just copy constructing breaks HashMap::GetAllocator due to const things, just use GetAddr
```

Comment:
You shouldn't copy construct it here. But please fix HashMap and remove the comment.

Reply:
Done